doc/tor.md

```diff
@@ -1,149 +1,289 @@
-*** Warning: Do not assume Tor support does the correct thing in Zcash; better Tor support is a future feature goal. ***
-
-TOR SUPPORT IN ZCASH
+Tor Support in Zcash
 ====================
 
-It is possible to run Zcash as a Tor hidden service, and connect to such services.
```

Comment:
This document is originally from upstream, and this rewrite is definitely going to cause backporting fun. I'd prefer that changes to documents from upstream be done in a separate commit, so it's easier to isolate during backporting.

Reply:
This change is now in its own commit.